packages/api/tests/controllers/user.test.js

```diff
@@ -37,3 +36,58 @@ describe('GET user Endpoint', () => {
     expect(res.statusCode).toBe(400);
   });
 });
+
+describe('POST user Endpoint', () => {
+  const user = {
+    email: 'test@test.com',
+    username: 'test_username',
+  };
+  let newUser;
+
+  beforeEach(async () => {
+    return db.User.create(user);
+  });
+
+  afterEach(async () => {
+    await db.User.destroy({ where: {} });
+  });
+
+  it('returns 201 when user is created', (done) => {
+    request(app)
+      .post(`/user`)
+      .set('Content-Type', 'application/json')
+      .send({ ...user, email: 'unique@email.com' })
+      .end((error, res) => {
+        expect(res.statusCode).toBe(201);
+        if (error) return done(error);
+        return done();
```

Comment:
Nit: I think you could just `return done(error);` in both cases here (and elsewhere). If `error` is `null` or `undefined`, it should have the same effect as passing nothing at all.